aarch64/bin/auth/disable

```diff
@@ -0,0 +1,5 @@
+#!/bin/rc
```

Comment:
Hmm; we need this for each arch? Surely this is a candidate for /rc/bin/auth and a 'bind -a /rc/bin/auth /bin/auth' ?